### PR TITLE
core: services: log_zipper: Use gzip over zip

### LINKED ISSUE #2161: We should avoid zip and start using gzip
It's simpler to handle and is a better abstraction over files than zip, zip was projected for folders.

### PR DESCRIPTION
**Pull request by patrickelectric**
This make it easier to process files
Fix #2161

### CODE REVIEW

File: core/services/log_zipper/main.py [hunk 1]
```diff
@@ -20,10 +20,11 @@
 
 
 def zip_files(files: List[str], output_path: str) -> None:
-    with zipfile.ZipFile(output_path, "w", zipfile.ZIP_DEFLATED) as zipper:
+
         for file in files:
             logger.debug(f"Zipping {file}...")
-            zipper.write(file)
+            with open(file, 'rb') as f_in, gzip.open(file + ".gz", 'wb') as f_out:
+                f_out.writelines(f_in)
```
Comment (joaoantoniocardoso) [inline core/services/log_zipper/main.py:27]: Weird indentation

Comment (patrickelectric) [reply core/services/log_zipper/main.py:27]: why ?

Comment (patrickelectric) [reply core/services/log_zipper/main.py:27]: The error in CI is from wrong black version, I'll rebase over master.

Comment (patrickelectric) [reply core/services/log_zipper/main.py:27]: @joaoantoniocardoso check now

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**